core/src/main/resources/data/upgrade_5.0.sql

```diff
@@ -2,4 +2,4 @@ UPDATE cpeecosystemcache SET ecosystem='MULTIPLE' WHERE vendor='tensorflow' AND
 
 UPDATE cpeecosystemcache SET ecosystem='MULTIPLE' WHERE vendor='scikit-learn' AND product='scikit-learn';
 
-UPDATE Properties SET value='5.1' WHERE ID='version';
\ No newline at end of file
+UPDATE Properties SET value='5.2' WHERE ID='version';
```

Comment:
The upgrade_5.0.sql should not be modified. Instead, a new file `upgrade_5.1.sql` should be created that contains the upgrade steps for a schema to go from 5.1 to 5.2.

For this PR we should revert the change:
```suggestion
UPDATE Properties SET value='5.1' WHERE ID='version';
```

Then create the new file with something like (note, I haven't tested this SQL yet): 
```sql
ALTER TABLE software ALTER COLUMN versionEndExcluding SET DATA TYPE VARCHAR(60);
ALTER TABLE software ALTER COLUMN versionEndIncluding SET DATA TYPE VARCHAR(60);
ALTER TABLE software ALTER COLUMN versionStartExcluding SET DATA TYPE VARCHAR(60);
ALTER TABLE software ALTER COLUMN versionStartIncluding SET DATA TYPE VARCHAR(60);

UPDATE Properties SET value='5.2' WHERE ID='version'; 
```

Reply:
That makes sense. I reverted the change to `upgrade_5.0.sql` and created the new file `upgrade_5.1.sql` according to your code.